rpc/detector/service.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package trivy.detector;
+option go_package = "detector";
+
+service OSDetector {
+    rpc Detect(OSDetectRequest) returns (DetectResponse);
+}
+
+message OSDetectRequest{
+    string os_family = 1;
+    string os_name = 2;
+    repeated Package packages = 3;
+}
+
+message DetectResponse{
+    repeated DetectedVulnerability detected_vulnerabilities= 1;
```

Comment:
I'd just call it `vulnerabilities`. The type is already defined in package `trivy.detector`

Reply:
Do you suggest like the following?

```suggestion
    repeated DetectedVulnerability vulnerabilities= 1;
```

Reply:
I fixed it